### PR TITLE
Lite: Tile operator test case updated

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
2 test case added to verify the correct flow of data post tiling.

Also some style check warnings are fixed.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @aselle: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by hgadig** [comment]
@aselle  Could you PTAL and approve.

File: tensorflow/lite/kernels/tile_test.cc [hunk 1]
```diff
@@ -115,61 +95,62 @@ TEST(TileTest, Float32HighDimension) {
...
+              ElementsAreArray(
+                  {11.f, 12.f, 13.f, 21.f, 22.f, 23.f, 11.f, 12.f, 13.f, 21.f,
+                   22.f, 23.f, 31.f, 32.f, 33.f, 41.f, 42.f, 43.f, 31.f, 32.f,
+                   33.f, 41.f, 42.f, 43.f, 11.f, 12.f, 13.f, 21.f, 22.f, 23.f,
+                   11.f, 12.f, 13.f, 21.f, 22.f, 23.f, 31.f, 32.f, 33.f, 41.f,
+                   42.f, 43.f, 31.f, 32.f, 33.f, 41.f, 42.f, 43.f}));
+  EXPECT_THAT(m.GetOutputShape(), ElementsAreArray({4, 4, 3}));
+}
+
+TEST(TileTest, Float32HighDimension2) {
```
Comment (srjoglekar246) [inline tensorflow/lite/kernels/tile_test.cc]: How are the new tests functionally different from the existing one? Is there a code path that wasn't being tested before? If yes, please rename these tests to reflect the same. `Float32HighDimension2` doesn't explain why we need this new case.

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]

File: tensorflow/lite/kernels/tile_test.cc (hunk 1 above)
Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/tile_test.cc]: @srjoglekar246 : Thanks for reviewing! You are absolutely right, it does not help in improving the coverage. But it helps another aspect of test case to show the behavior in details. Actually i observed that when one of the input dim or multiplier is 1, then the transformation is not quite clear from the output(code reading). So i added these 2 test cases, to show better depiction of transformation in output. Please provide your valuable opinion. Once i get your opinion on this, will change the name as well. Thanks!

Comment (srjoglekar246) [reply tensorflow/lite/kernels/tile_test.cc]: IMO unit tests aren't the best place to showcase behavior. Ideally our documentation should be clear on what the expected inputs/outputs are. We wouldn't want to bloat our unit tests if they aren't increasing coverage.

Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/tile_test.cc]: @srjoglekar246: Thanks for your feedback. Sometimes these kind of test cases, helps preventing any unwanted change in core logic. But I will go by your conclusion. Please close the PR if you feel it's not required. Thanks a lot for your effort in reviewing.

**Pull request closed without merging**